sdk/src/main/java/com/mapbox/maps/MapView.kt

```diff
@@ -296,9 +301,54 @@ open class MapView : FrameLayout, MapPluginProviderDelegate, MapControllable {
    */
   @SuppressLint("ClickableViewAccessibility")
   override fun onTouchEvent(event: MotionEvent): Boolean {
+    interceptedActionDownEvent?.let {
+      try {
+        return mapController.onTouchEvent(it)
+      } finally {
+        it.recycle()
+        interceptedActionDownEvent = null
+      }
+    }
+    interceptedActionPointerDownEvent?.let {
+      try {
+        return mapController.onTouchEvent(it)
+      } finally {
+        it.recycle()
+        interceptedActionPointerDownEvent = null
+      }
+    }
     return mapController.onTouchEvent(event)
   }
 
+  override fun onInterceptTouchEvent(event: MotionEvent): Boolean {
+    return when (event.actionMasked) {
+      MotionEvent.ACTION_DOWN -> {
+        interceptedActionDownEvent = MotionEvent.obtain(event)
+        false
+      }
+      MotionEvent.ACTION_POINTER_DOWN -> {
+        interceptedActionPointerDownEvent = MotionEvent.obtain(event)
```

Comment:
Since there's no difference how `interceptedActionDownEvent` and `interceptedActionDownEvent` are handled - maybe just add them to a collection? 
Also - seems case with 3+ fingers is not intercepted correctly. Zoom in-out works fine using 3 and more fingers if the gesture is started on the map, and not when it starts on view annotation. Suppose its due to the fact that we overwrite `interceptedActionPointerDownEvent` on 3 and following events

Reply:
> Since there's no difference how interceptedActionDownEvent and interceptedActionDownEvent are handled - maybe just add them to a collection?

What is the purpose of such an approach?

> Suppose its due to the fact that we overwrite interceptedActionPointerDownEvent on 3 and following events

Yeah, it's a bug. Also try to store `interceptedActionPointerDownEvent` when it's ONLY empty. But unsuccessfully

Reply:
> What is the purpose of such an approach?

It will be less code, especially if we'll take into account 3-4-... multitouch events